### PR TITLE
feat(payment-stripe): update payment customer data events handle

### PR DESCRIPTION
**Pull request by OlegDO**
Unit tests infra for Stripe webhook handlers
Update delete external acc find params

### CODE REVIEW

File: microservices/payment-stripe/src/services/payment-gateway/stripe.ts [hunk 1]
```diff
@@ -1025,6 +1025,13 @@ class Stripe extends Abstract {
     if (!this.isExternalAccountIsBankAccount(externalAccount)) {
       const card = await this.getCardById(externalAccount.id);
 
+      if (!card) {
+        throw new BaseException({
+          status: 500,
+          message: messages.getNotFoundMessage('Failed to handle external account updated. Card'),
+        });
```
Comment (MatthewPattell) [inline microservices/payment-stripe/src/services/payment-gateway/stripe.ts:1032]:
```suggestion
          message: messages.getNotFoundMessage('Card: Failed to handle external account updated.'),
        });
```

File: microservices/payment-stripe/src/services/payment-gateway/stripe.ts [hunk 2]
```diff
@@ -1049,6 +1056,15 @@ class Stripe extends Abstract {
 
     const bankAccount = await this.getBankAccountById(externalAccount.id);
 
+    if (!bankAccount) {
+      throw new BaseException({
+        status: 500,
+        message: messages.getNotFoundMessage(
+          'Failed to handle external account updated. Bank account',
```
Comment (MatthewPattell) [inline microservices/payment-stripe/src/services/payment-gateway/stripe.ts:1063]: the same

File: microservices/payment-stripe/src/services/payment-gateway/stripe.ts [hunk 3]
```diff
@@ -1148,24 +1164,40 @@ class Stripe extends Abstract {
...
-    const externalAccountId = this.extractId(externalAccount.account);
+    const externalAccountId = this.extractId(externalAccount.id);
 
     if (!this.isExternalAccountIsBankAccount(externalAccount)) {
       const card = await this.getCardById(externalAccountId);
 
+      if (!card) {
+        throw new BaseException({
+          status: 500,
+          message: messages.getNotFoundMessage('Failed to handle external account deleted. Card'),
```
Comment (MatthewPattell) [inline microservices/payment-stripe/src/services/payment-gateway/stripe.ts:1175]: the same

File: microservices/payment-stripe/src/services/payment-gateway/stripe.ts [hunk 4]
```diff
@@ -1148,24 +1164,40 @@ class Stripe extends Abstract {
...
       return;
     }
 
     const bankAccount = await this.getBankAccountById(externalAccountId);
 
+    if (!bankAccount) {
+      throw new BaseException({
+        status: 500,
+        message: messages.getNotFoundMessage(
+          'Failed to handle external account deleted. Bank account',
```
Comment (MatthewPattell) [inline microservices/payment-stripe/src/services/payment-gateway/stripe.ts:1190]: the same

**Review by MatthewPattell — APPROVED** [review]

File: microservices/payment-stripe/src/services/payment-gateway/stripe.ts (hunk 1 above)
Comment (OlegDO) [reply microservices/payment-stripe/src/services/payment-gateway/stripe.ts:1032]: getNotFoundMessage will add postfix 
image

**Pull request merged**